### PR TITLE
example config file updates

### PR DESCRIPTION
**Pull request by mmnsgo**
removed deprecated variables
```
    "catch_randomize_reticle_factor": 0.7,
    "catch_randomize_spin_factor": 0.9
```

add new settings for catch/throw
```
    "catch_throw_parameters": {
      "excellent_rate": 0.1,
      "great_rate": 0.5,
      "nice_rate": 0.3,
      "normal_rate": 0.1,
      "spin_success_rate" : 0.6
    }
```

set  ``` "health_record": false ``` as there's no documentation regarding what is sent, where or how often.

### CODE REVIEW

File: configs/config.json.cluster.example [hunk 1]
```diff
@@ -94,19 +94,24 @@
     "action_wait_max": 4,
     "debug": false,
     "test": false,
-    "health_record": true,
+    "health_record": false,
     "location_cache": true,
```
Comment (TheSavior) [inline configs/config.json.cluster.example]: Please leave this to true

File: configs/config.json.example [hunk 2]
```diff
@@ -110,12 +110,10 @@
     "action_wait_max": 4,
     "debug": false,
     "test": false,
-    "health_record": true,
+    "health_record": false,
```
Comment (TheSavior) [inline configs/config.json.example]: On all of these please.

File: configs/config.json.cluster.example (hunk 1 above)
Comment (mmnsgo) [inline configs/config.json.cluster.example]: what does it do? what data does it send? where? how often?

File: configs/config.json.example (hunk 2 above)
Comment (mmnsgo) [inline configs/config.json.example]: what does it do? what data does it send? where? how often?

Comment (nivong) [inline configs/config.json.example]: you should read it send analytics information

Comment (TheSavior) [inline configs/config.json.example]: We explain this very well in the readme.

**Pull request closed without merging**